app/views/catalog/_field_list.html.erb

```diff
@@ -1,10 +1,22 @@
-<dl>
-  <% fields.each do |method| 
-    asset.send(method).tap do |value| 
-      if !value.empty? %>
-        <dt><%= method.gsub('_',' ').gsub(/clip /i, '') %></dt>
-        <dd><%= value %></dd><% 
-      end
-    end
-  end %>
-</dl>
\ No newline at end of file
+<%
+# default to 1 column, and only allow up to 4
+columns ||= 1
+columns = [columns.to_i, 4].min
```

Comment:
i'm not a fan of checking param values or setting defaults in the partial. The parameterization is a good thing, but a view is not a method, so it gets awkward when we treat it like one. But until we come up with a better way, I'll fail explicitly here on invalid number of columns.